plotly/plotly/plotly.py

```diff
@@ -1271,6 +1334,50 @@ def validate_credentials(credentials):
         raise exceptions.PlotlyLocalCredentialsError()
 
 
+def add_share_key_to_url(response, **payload):
+    """
+    Update plot's url to include the secret key
+
+    payload keyword agruments:
+    platform ('python') -- the platform that is associated with this url
+    version (string) -- the Plotly version that was used to create the plot
+    args (dict) -- contains the data from figure
+    un (string) -- username from credentials file
+    key (string) -- api_key from credentials file
+    origin('plot') --
```

Comment:
no but it's part of `payload` arguments,   should I just put the ones we use in `add_share_key_to_url`?